### PR TITLE
modified screenshots and sorting of documentation

### PR DESCRIPTION
**Pull request by albertogilf**
Updated screenshots and the order of the documentation

### CODE REVIEW

File: docs/chemwalker_doc.md [hunk 1]
```diff
@@ -24,7 +24,19 @@ The fields are presented in the image. Bold fields are mandatory:
...
+The information from the molecular network is necessary to annotate the nodes (features). For that, chemwalker needs the job id of the molecular network task. To view the task_id, check the red rectangle in the [image](./img/workflows/task_id.png). The task id can correspond to a [GNPS2 Feature based molecular networking (FBMN)](https://gnps2.org/workflowinput?workflowname=feature_based_molecular_networking_workflow), to a [GNPS FBMN](https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22FEATURE-BASED-MOLECULAR-NETWORKING%22,%22library_on_server%22:%22d.speclibs;%22%7D) or a [GNPS Molecular Networking v2](https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22METABOLOMICS-SNETS-V2%22,%22library_on_server%22:%22d.speclibs;%22%7D). 
+
+
+### Example Input
+
+| #   | Information                                        | Value                                       |
+| --- | -------------------------------------------------- | ------------------------------------------- |
+| 1.  | Task description                                  | empty                                       |
```
Comment (mwang87) [inline docs/chemwalker_doc.md:34]: This default shoudl be better than empty. Something that gives people a sense of what they should put there, e.g. Chemwalker Analysis of GNPS2 Network

File: docs/chemwalker_doc.md [hunk 2]
```diff
@@ -24,7 +24,19 @@ The fields are presented in the image. Bold fields are mandatory:
...
+| 2.  | **GNPS2, FBMN, or FBMNv2 task id**                | b6d12e1e42b64ad6b20e7d38d5a4214b           |
+| 3.  | **Workflow type (GNPS2, FBMN or V2)**             | GNPS2                                       |
+| 4.  | **Component (Molecular family) index**            | 11                                          |
+| 5.  | **File selection**                                | COCONUT.tsv                                 |
```
Comment (mwang87) [inline docs/chemwalker_doc.md:38]: Is this correct? This is not a valid path in GNPS2.

File: docs/chemwalker_doc.md (hunk 1 above)
Comment (albertogilf) [reply docs/chemwalker_doc.md]: Set a description of the field.

File: docs/chemwalker_doc.md (hunk 2 above)
Comment (albertogilf) [reply docs/chemwalker_doc.md]: Set full path of the database instead of only the name.

**Pull request merged**